### PR TITLE
BookStore

### PR DESCRIPTION
**Pull request by brahimdidi**
The Bookstore is a website similar to the "Awesome Books" website built in the previous module  that allows you to:
Display a list of books, Add a book, Remove a selected book.
## Requierments 
Initialise React app.
The building blocks of your app should be set as re-usable components. Create a directory for your components.
Add [React Router](https://reactrouter.com/web/guides/quick-start) and set two <Route>s and <Link>s for the app's navigation:
Books - the default view:
Should display the list of books (empty at this point but it should be ready for the data) with the Remove button (no funcionality yet).
Create a component called Book for displaying a single book (receive a title and an author as prop) and reuse it in a component that displays a list of books.
Should have a form for adding a book (no functionality yet).
Create a separate component for this form (with inputs for a title and an author [IMPORTANT in the design you can see an input for a category - please replace it with an author.]).
Categories:
Should display a button "Check status" only.
Styling is not required at this point.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,49 @@
...
+A text editor(preferably Visual Studio Code) Install Git Node Usage Clone this repository git clone git@github.com:brahimdidi/math-magic.git cd math-magic Run project npm start Open page in browser $ open dist/index.html Author bust_in_silhouette Brahim
+
+## GitHub:
+ @brahimdidi Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support Give a star if you like this project!
+
+memo License This project is MIT licensed.
```
Comment (CollinsTatang) [inline README.md]: - [x] Kindly update your readme file following the [Readme Template](https://github.com/microverseinc/readme-template/blob/master/README.md) and add all main headers from the template.

File: src/components/AddBook.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+const AddBook = () => (
+  <section className="addbook-section flex flex-column">
+    <h1>Add new book</h1>
+    <form className="form">
+      {/* <label id="title"> */}
+      <input className="title" placeholder="title" name="title" />
+      {/* </label> */}
+      {/* <label id="author"> */}
+      <input className="author" placeholder="author" name="author" />
+      {/* </label> */}
```
Comment (CollinsTatang) [inline src/components/AddBook.js]: - [x] Please remove all commented code from your code base to make it clean and easy to maintain.

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @brahimdidi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (brahimdidi) [reply README.md]: done

File: src/components/AddBook.js (hunk 2 above)
Comment (brahimdidi) [reply src/components/AddBook.js]: done

**Pull request merged**